src/translations/en.json

```diff
@@ -60,5 +60,7 @@
   "edit": "Edit",
   "like": "Like",
   "comment": "Comment",
-  "by": "by"
+  "by": "by",
+  "page_forbidden": "Page Requires Authentication",
```

Comment:
It does not give error it gives warning and we should not add english translation in all files. warning would be helpful in reminding the missing translation.